lib/cards/KCardGrid.vue

```diff
@@ -1,44 +1,46 @@
 <template>
 
-  <div
-    v-if="showGrid"
-    class="k-card-grid"
-  >
-    <transition
-      name="fade"
-      mode="out-in"
-      appear
+  <div>
```

Comment:
I had to add a div here so we can always have at least one element, even if `showGrid` is false, if on mount we dont have an element in the root of the component, `useKResponsiveElement` will fail.

Reply:
That sound fine, we will just need to test carefully if it doesn't cause some unexpected trouble with loading skeleton experience, particularly on slower networks. `showGrid` function has more details in comment about what issues it takes care of.